web/dev.ts

```diff
@@ -817,6 +820,47 @@ export async function setupDevDB() {
         await entityManager.save(schools[i]);
         console.log("Inserted Dev School " + i);
     }
+
+    //Insert expert data
+    const expertiseTags: ExpertiseTag[] = [];
+
+    const tag1 = new ExpertiseTag();
+    tag1.name = "LTE";
+
+    expertiseTags.push(tag1);
+
+    const tag2 = new ExpertiseTag();
+    tag2.name = "Glasfaser";
+
+    expertiseTags.push(tag2);
+
+    for (let i = 0; i < expertiseTags.length; i++) {
+        await entityManager.save(expertiseTags[i]);
+        console.log("Inserted Expertise Tag " + i);
+    }
+
+    const experts: ExpertData[] = [];
+
+    const expert1 = new ExpertData();
+    expert1.student = students[5];
+    expert1.contactEmail = "contact@jufo-tufo.de";
+    expert1.description = "JuFo is great!";
+
+    experts.push(expert1);
+
+    const expert2 = new ExpertData();
+    expert2.student = students[6];
+    expert2.contactEmail = "contact@jufo-tufo.de";
```

Comment:
We can discuss, whether we allow the same contact email address for several experts. But I think it might have its use cases.